<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwMappingWidgetUi</class>
 <widget class="QWidget" name="AwMappingWidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>444</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="AwMapWidget" name="qvtkWidget3D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>350</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AwTopoWidget" name="topoWidget2D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>350</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="button2D">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>2D</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button3D">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>3D</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonBoth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Both</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSensors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Show sensors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxLabels">
         <property name="text">
          <string>Show Labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxContours">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Show contours</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Colormap:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AwComboCMap" name="comboCMap"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AwTopoWidget</class>
   <extends>QWidget</extends>
   <header location="global">widget/AwTopoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AwComboCMap</class>
   <extends>QComboBox</extends>
   <header location="global">widget/AwComboCMap.h</header>
  </customwidget>
  <customwidget>
   <class>AwMapWidget</class>
   <extends>QWidget</extends>
   <header location="global">widget/AwMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button3D</sender>
   <signal>clicked(bool)</signal>
   <receiver>AwMappingWidgetUi</receiver>
   <slot>to3D(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoth</sender>
   <signal>clicked(bool)</signal>
   <receiver>AwMappingWidgetUi</receiver>
   <slot>showBoth(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2D</sender>
   <signal>clicked(bool)</signal>
   <receiver>AwMappingWidgetUi</receiver>
   <slot>to2D(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showSensors(bool)</slot>
  <slot>to3D(bool)</slot>
  <slot>showBoth(bool)</slot>
  <slot>to2D(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
